tools/js2c.py

```diff
@@ -310,7 +310,7 @@ def JS2C(source, target):
     else:
       ids.append((id, len(lines)))
 
-    escaped_id = id.replace('/', '$')
+    escaped_id = id.replace('/', '_')
```

Comment:
For posterity, it's mentioned in the commit log: sigils in identifiers are not supported on AIX.